tabular/src/autogluon/tabular/models/rf/rf_model.py

```diff
@@ -224,6 +334,7 @@ def _fit(self,
             model.estimators_ = None
         self.model = model
         self.params_trained['n_estimators'] = self.model.n_estimators
+        self.compile(path=self.path)
```

Comment:
Are we compiling twice?? Once at end of fit and once during save? Why?

Reply:
Removed this entry here. 

NOTE: Actually the second trigger to the compile function would simply return the precompiled predictor.